builder/js/ncf.js

```diff
@@ -588,60 +585,50 @@ $scope.groupMethodsByCategory = function () {
   $scope.checkDeprecatedFilter = function(methods){
     return methods.some(function(method){return method.deprecated === undefined });
   }
+
   $scope.checkFilterCategory = function(methods){
```

Comment:
At least that should be applied on the method!

Nevermind, we will work this out in another PR